INSTALL.md

```diff
@@ -36,9 +54,9 @@ This will create a folder named 'ilios' in your server root directory.  The enti
 cd ilios
 ```
 You should now be in the '/web/ilios3/ilios'  
```

Comment:
Probably should read "You should now be in the '/web/ilios3/ilios'  ***directory***"